content/docs/1.6.0/advanced-resources/os-distro-specific/talos-linux-support.md

```diff
@@ -60,6 +60,40 @@ For detailed instructions, see the Talos documentation on [Editing Machine Confi
 
 - Exclusive to v1 data volume: currently, within a Talos Linux cluster, Longhorn only supports v1 data volume. The v2 data volume isn't currently supported in this environment.
 
+## Talos Linux Upgrades
+
+When [upgrading a Talos Linux node](https://www.talos.dev/v1.7/talos-guides/upgrading-talos/#talosctl-upgrade), always include the `--preserve` option in the command. This option explicitly tells Talos to keep ephemeral data intact.
+
+Example:
+
+```
+talosctl upgrade --nodes 10.20.30.40 --image ghcr.io/siderolabs/installer:v1.7.6 --preserve
+```
+
+> **Caution:**
+> If you do not include the `--preserve` option, Talos wipes `/var/lib/longhorn`, destroying all replicas stored on that node.
+## Recovering from an Upgraded Node without Preserving Data
```

Comment:
```suggestion
> **Caution:**
> If you do not include the `--preserve` option, Talos wipes `/var/lib/longhorn`, destroying all replicas stored on that node.

## Recovering from an Upgraded Node without Preserving Data
```
And also make changes to the other versions.

Reply:
Corrected `##` to `###`